packages/neuron-ui/src/components/Overview/index.tsx

```diff
@@ -191,33 +199,46 @@ const Overview = () => {
 
   return (
     <PageContainer head={t('navbar.overview')} notice={pageNotice} isHomePage>
-      <div className={styles.mid}>
-        <div className={styles.balance}>
-          <span className={styles.balanceTitle}>
-            {t('overview.balance')}
-            {showBalance && <EyesOpen onClick={onChangeShowBalance} className={styles.balanceIcon} />}
-            {!!showBalance || <EyesClose onClick={onChangeShowBalance} className={styles.balanceIcon} />}
-          </span>
-          {showBalance ? (
-            <CopyZone content={shannonToCKBFormatter(balance, false, '')} className={styles.copyBalance}>
-              <span className={styles.balanceValue}>{shannonToCKBFormatter(balance)}</span>
-            </CopyZone>
-          ) : (
-            <span className={styles.balanceValue}>{HIDE_BALANCE}</span>
-          )}
-          <span className={styles.balanceUnit}>CKB</span>
-          <BalanceSyncIcon connectionStatus={connectionStatus} syncStatus={syncStatus} />
-          <BalanceRight className={styles.backgroundImg} />
+      <div className={styles.topContainer}>
+        <div className={styles.mid}>
+          <div className={styles.balance}>
+            <span className={styles.balanceTitle}>
+              {t('overview.balance')}
+              {showBalance && <EyesOpen onClick={onChangeShowBalance} className={styles.balanceIcon} />}
+              {!!showBalance || <EyesClose onClick={onChangeShowBalance} className={styles.balanceIcon} />}
```

Comment:
Use ternary to return `null` explicitly, and it's consistent with the code style used below(at line 311, 312)